test/test_tree_2d_advection.jl

```diff
@@ -12,8 +12,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
   @trixi_testset "elixir_advection_basic.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_basic.jl"),
       # Expected errors are exactly the same as in the parallel test!
-      l2   = [8.311947673061856e-6],
-      linf = [6.627000273229378e-5],
+      l2   = [8.312144636192947e-6],
+      linf = [6.626823860056419e-5],
```

Comment:
Related to the comment of @sloede above: Please use another example to demonstrate this.